### DISCUSSION TITLE
Stream Call Audio

### DISCUSSION THREAD

**Discussion by gavinsmith8181** [rtckit/eqivo#22] [Ideas]
Hey! I saw on Twilio and Signalwire that call audio can be streamed over, was wondering if that's possible on Eqivo as well, and if so how can i do it? 

it can be a great option to do, since i want to also stream the call to agents in real time.

**Comment by cdosoftei** [comment]
Hello!

I believe you're referring to the streaming over WebSockets features, is that correct? If so, the feature is planned, there are some interesting FreeSWITCH modules which can be used as a building block.

However, I must ask about the use case; streaming audio over WebSockets is typically employed for Speech-to-Text purposes (specifically for custom implementations).

Text-to-Speech is already available in Eqivo via the [GetSpeech](https://eqivo.org/#getspeech) RestXML element.

Can you elaborate on the streaming calls to agents use case?

**Reply by gavinsmith8181** [reply]
@cdosoftei

**Comment by gavinsmith8181** [comment]
Hey! The use case would be mainly to allow coach other agents / coach a specific agent.

BTW is there any way else to talk to beside github? would love to talk also on discord / telegram.

**Reply by cdosoftei** [reply]
`<Conference>`  works best for monitoring and coaching too; check out the `ghost` member flag for [mod_conference](https://developer.signalwire.com/freeswitch/FreeSWITCH-Explained/Modules/mod_conference_3965534/) for privileged users (i.e. they are not being advertised as members but can behave as such).

Regarding other discussion platforms, I've just set up a [Zulip chat space](https://rtckit.zulipchat.com/) (then navigate to the [#eqivo channel](https://rtckit.zulipchat.com/#narrow/stream/414323-eqivo)) -- see you there!
